docs/specs/event.md

```diff
@@ -172,6 +179,187 @@ Occurs right before the account anonymization is scheduled.
 }
 ```
 
+#### authentication.pre_initialize
+
+Occurs right before any authentication, such as login.
+
+Fields in payload:
+- `authentication_context`: An [`AuthenticationContext`](./event_models.md#authenticationcontext) object.
+
+Example payload:
+
+```json5
+{
+  "payload": {
+    "authentication_context": {
```

Comment:
I've renamed authentication to `authentication_context` to avoid naming collision with `authentication`